tests/epics/test_signals.py

```diff
@@ -332,6 +340,44 @@ async def test_pva_table(ioc: IOC) -> None:
             q.close()
 
 
+async def test_pvi_structure(ioc: IOC) -> None:
+    if ioc.protocol == "ca":
+        # CA can't do structure
+        return
+    # Make and connect the backend
+    backend = await ioc.make_backend(Dict[str, Any], "pvi")
+
+    # Make a monitor queue that will monitor for updates
+    q = MonitorQueue(backend)
+
+    expected = {
+        "pvi": {
+            "width": {
+                "rw": f"{PV_PREFIX}:{ioc.protocol}:width",
+            },
+            "height": {
+                "rw": f"{PV_PREFIX}:{ioc.protocol}:height",
+            },
+        },
+        "record": {
+            "_options": {
+                "atomic": ANY,  # the field can be true or false
+                "queueSize": 0,
+            },
+        },
```

Comment:
I think we don't care about any of the record specific bits

```suggestion
        "record": ANY
```